modules/microshift-storage-backup-volume-snapshots.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assemblies:
+//
+// microshift/volume-snapshots-microshift.adoc
+
+:_content-type: PROCEDURE
+[id="backing-up-a-volume-snapshotting_{context}"]
+= Backing up a volume snapshot
+
+Snapshots of {product-title} data are created as read-only logical volumes (LVs) located on the same devices as the original data. All local volumes must be manually mounted before they can be copied as persistent volumes (PVs) and used as backup copies. To use a snapshot of a {product-title} storage volume as a backup, you can locate it on the local host and then move it to a secure location. To find specific snapshots and copy them, use the following procedure.
+//are these LVs or PVs?
+
+.Prerequisites
+
+. You have root access to the host machine.
+. A {product-title} snapshot was successfully created.
+
+.Procedure
+
+. Get the `volumeSnapshot.status.volumeSnapshotContentName` by using the following command syntax:
++
+[source, terminal]
+----
+$ oc get volumesnapshot -n test-ns my-snap -o yaml
+----
+
+. Then, get the `volumeSnapshotContent.status.snapshotHandle` by using the following command syntax:
++
+[source, terminal]
+----
+$ oc get volumesnapshotcontent snapcontent-<retrieved volume identity> -o yaml
+----
+
+. Display the snapshots so you can determine the one you want to backup by using the following command syntax:
++
+[source, terminal]
+----
+$ sudo lvdisplay rhel/732e45ff-f220-49ce-859e-87ccca26b14c
+----
++
+.Example output
+
+[source, terminal]
+----
+--- Logical volume ---
+LV Path                /dev/rhel/732e45ff-f220-49ce-859e-87ccca26b14c
+LV Name                732e45ff-f220-49ce-859e-87ccca26b14c
+VG Name                rhel
+LV UUID                6Ojwc0-YTfp-nKJ3-F9FO-PvMR-Ic7b-LzNGSx
+LV Write Access        read only
+LV Creation host, time rhel-92.lab.local, 2023-08-07 14:45:26 -0500
+LV Pool name           thinpool
+LV Thin origin name    a2d2dcdc-747e-4572-8c83-56cd873d3b07
+LV Status              available
+# open                 0
+LV Size                1.00 GiB
+Mapped size            1.04%
+Current LE             256
+Segments               1
+Allocation             inherit
+Read ahead sectors     auto
+- currently set to     256
+Block device           253:11
+----
+
+//would this be the same as restoring a volume? by specifying the `VolumeSnapshot` object as the `dataSource` in a `PersistentVolumeClaim`
+. Mount the volumes by specifying the `VolumeSnapshot` object as the `dataSource` in a `PersistentVolumeClaim`
++
+[source, terminal]
+----
+$ sudo
+----
+
+. Copy the snapshot to a secure location by using the following command syntax:
```

Comment:
Without knowing what is in the filesystem, we can't give them a precise command to run here. They know their data. It should be enough to tell them to copy the files from the mounted filesystem. If not, a generic command would just be something like `sudo cp -r /mnt/snapshot <destination>`.